tests/framework/extensions/namespaces/getnamespaces.go

```diff
@@ -0,0 +1,22 @@
+package namespaces
+
+import (
+	"sort"
+
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+)
+
+func GetSortedNamespaces(steveclient *v1.Client) ([]string, error) {
```

Comment:
Public functions need their own godoc comment explanation. Could we add an explanation?